packages/app/components/search/search-status-bar.tsx

```diff
@@ -99,10 +99,10 @@ const SearchStatusBar = ({
                             value={searchOptions.filter}
                             onChange={e => onFilterChange(e.target.value)}
                         >
-                            <option value=""></option>
+                            <option value="">Show All States</option>
 
                             {states.map(state => (
-                                <option key={state} value={`state:${state}`}>
+                                <option key={state} value={`state:"${state}"`}>
```

Comment:
Lucene requires queries that contain spaces to be contained in quotation marks. Quotation marks do not harm queries that do not require it, so this PR adds them by default.